src/js/main.js

```diff
@@ -102,53 +91,27 @@ let completedLine = (() => {
 })
 
 let newLevel = (() =>{
-    
-    if (totalLinesCleared===level*10 && totalLinesCleared>0){
-        level++;
-        switch(level){
-            case 1:
-                speed=1000
-                break;
-            case 2:
-                speed=500
-                break;
-            case 3:
-                speed=400
-                break;
-            case 4:
-                speed=300
-                break;
-            case 5:
-                speed=250
-                break;
-            case 6:
-                speed=200
-                break;
-            case 7:
-                speed=150
-                break;
-            case 7:
-                speed=100
-                break;
-            case 8:
-                speed=90
-                break;
-            case 9:
-                speed=80
-                break;
-            case 10:
-                speed=70
-                break;
-            case 11:
-                speed=60
-                break;
-            case 12:
-                speed=50
-                break;
+    const speedForLevel = [
```

Comment:
Perso je suis chaude de le laisse avec le switch je trouve que c'est plus lisible eheh

Reply:
et que penses-tu d' un middle-ground comme ca ?
`function speedForLevel(level) {
    switch(level){
        case 1: return 1000
        case 2: return 500
        case 3: return 400
        case 4: return 300
        case 5: return 250
        case 6: return 200
        case 7: return 150
        case 7: return 100
        case 8: return 90
        case 9: return 80
        case 10: return 70
        case 11: return 60
        default: return 50
    }
}

let newLevel = (() => {
    if (totalLinesCleared >= level * 10 && totalLinesCleared > 0) { 
        if (level < speedForLevel.length) {
            speed = speedForLevel(level)
            level++
        }
    }
})`

Reply:
I made the changes, back to switch logic